### PR TITLE
fixed linking and formatting issues

### PR DESCRIPTION
**Pull request by angelinetu**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
added styles in styles.ts for the image, text, comments, etc.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ronniebeggs

### CODE REVIEW

File: src/screens/PostScreen.tsx [hunk 1]
```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
...
-      <Text>September 19</Text>
-      <Text>
+      {/* Profile Header */}
+      <View style={styles.profileHeader}>
+        <ProfilePlaceholder style={styles.profileIcon} />
+        <View>
+          <Text style={styles.profileName}>rbeggs</Text>
+          <Text style={styles.postDate}>September 19</Text>
+        </View>
+      </View>
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: Given that we want the username + profile placeholder to be grouped together, let's wrap them in their own div. Then apply `justifyContent: 'space-between'` to the `profileHeader` style to spread information across the width of the screen.

File: src/screens/PostScreen.tsx [hunk 2]
```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
 export default function PostScreen() {
   return (
     <View style={styles.container}>
-      <ProfilePlaceholder />
-      <Text>rbeggs</Text>
-      <Text>September 19</Text>
-      <Text>
+      {/* Profile Header */}
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: Consider applying styles to add spacing between the content and the sides of the screen. You can do so with the `padding` prop applied to the `container` View.

File: src/screens/PostScreen.tsx [hunk 3]
```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
...
+          uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+        }}
+      />
+
+      {/* Like and Share Section */}
+      <View style={styles.interactionRow}>
+        <HeartIcon style={styles.icon} />
+        <Text style={styles.likesText}>256 Likes</Text>
+        <ShareIcon style={styles.icon} />
+      </View>
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: Apply similar styles as the `profileHeader` suggestion to spread icons across the width of the screen.

File: src/screens/PostScreen.tsx [hunk 4]
```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
...
+
+      {/* Comments */}
+      <View style={styles.commentSection}>
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx:53]: One way to add a content divider would be to apply a border style to the top of the `commentSection` div.

File: src/screens/PostScreen.tsx [hunk 5]
```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
...
+        <View style={styles.comment}>
+          <ProfilePlaceholder style={styles.commentIcon} />
+          <View>
+            <Text style={styles.commentName}>philip_ye</Text>
+            <Text style={styles.commentText}>
+              This organization is doing amazing work tackling the complex root
+              causes of the issue.
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: How could we integrate the comment text here? Try placing the username and date text in the same div, that way they appear inline, above the comment text.

File: src/screens/styles.ts [hunk 6]
```diff
@@ -7,4 +7,71 @@ export const styles = StyleSheet.create({
...
+    fontWeight: 'bold',
+    fontSize: 16,
+  },
+  postDate: {
+    color: 'gray',
+  },
+  postText: {
+    fontSize: 14,
+    lineHeight: 20,
+    marginBottom: 10,
```
Comment (ronniebeggs) [inline src/screens/styles.ts]: Let's avoid applying spacing styles to text.

File: src/screens/styles.ts [hunk 7]
```diff
@@ -7,4 +7,71 @@ export const styles = StyleSheet.create({
...
+    height: 30,
+    marginRight: 10,
+  },
+  commentName: {
+    fontWeight: 'bold',
+  },
+  commentText: {
+    fontSize: 14,
+    lineHeight: 18,
+  },
```
Comment (ronniebeggs) [inline src/screens/styles.ts]: When we define absolute widths and margins and place them in line with each other, they'll compete and push content off the side of the screen. Using flex boxes with `rowGap` and `columnGap` properties can remedy this issue.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Looking good so far! I've left a few comments to consider before tomorrow's workshop. See if you can utilize more flexbox properties to achieve our desired styles (practice this here: https://flexboxfroggy.com/)
